bin/extract_to_mediawiki

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env php
+<?php
+
+$language = '';
+switch( $argv[1]?? '' ) {
+	case 'de':
+		$language = 'de_DE';
+		break;
+	case 'en':
+		$language = 'en_GB';
+		break;
+	default:
+		die( "You must provide a language (de | en )!\n");
+}
+
+$contents = file_get_contents(__DIR__ ."/../i18n/$language/data/use_of_funds_content.json" );
+$json = json_decode( $contents );
+$encodedContent = str_replace(
+	"'",
+	'&#039;',
+	htmlspecialchars(
+		json_encode( $json ),
+		ENT_NOQUOTES,
```

Comment:
This breaks when MediaWiki changes our single quotes to double quotes, so unfortunately we'll have to use `ENT_QUOTES` here